manifests/config/setting.pp

```diff
@@ -21,15 +21,13 @@
 define php::config::setting(
   $key,
   $value,
-  $file,
+  String $file,
```

Comment:
This should probably be a `Stdlib::AbsolutePath` since it is passed to a `path` parameter.